### ISSUE TITLE
Search Bar Feature Request: Allow search query to be edited as plain text

### ISSUE THREAD

**Issue by malwilley** [getsentry/sentry#82395]
### Problem Statement

The new search UI has improved feature discovery and new user adoption, but also presents usability issues for power users who are aware of Sentry's search syntax and would otherwise be able to edit queries more quickly.

Some parts of the query editing are slower with the new UI, such as:

- If you want to negate a filter, you have to click "is" to change to "is not" after creating the filter. With text, you could write out `!is:unresolved`
- If you want to change a filter key or value, you need to click (or arrow over and press enter) to enter edit mode. With text you can immediately go to the part you wish to edit.

### Solution Brainstorm

The search bar could support two "modes" - the current tokenized mode, and a plain text mode. Users could toggle between them, and that preference would be stored somewhere to remember the selection. The plain text mode would still need to support autocomplete features to make it usable.

### Product Area

Issues

**Comment by nipunh** [comment]
@malwilley Can i work on this issue?

**Comment by malwilley** [comment]
@nipunh this is a pretty large feature which still requires some product and design input before any work can be done on it. I appreciate the eagerness, but perhaps it would be better to focus on another issue!
